test/sequential/test-inspector-stop-profile-after-done.js

```diff
@@ -15,13 +15,22 @@ async function runTests() {
                                  }, 10);`);
   const session = await child.connectInspectorSession();
 
+  let stderrString = await child.nextStderrString();
+  while (!stderrString.includes('Debugger listening on')) {
+    stderrString += await child.nextStderrString();
+  }
+
   session.send([
     { 'method': 'Profiler.setSamplingInterval', 'params': { 'interval': 100 } },
     { 'method': 'Profiler.enable' },
+    { 'method': 'Profiler.start' },
```

Comment:
I believe we should maintain the order. My concern is there might be a genuine bug if this change fixes the test failure. Chrome DevTools (and I suppose other frontends) will fire Profile.start long after Runtime.runIfWaitingForDebugger (runIfWaiting... is sent to start the target that is waiting for the frontend to attach while Profile.start is started at a later point, e.g. when the user clicks a button to start profiling)

Reply:
OK, I put it back. Thanks.